connection.js

```diff
@@ -211,7 +211,6 @@ class Connection {
         this.set('hello', 'host', undefined);
         this.set('tls',   'enabled', true);
         for (const t of ['cipher','verified','verifyError','peerCertificate']) {
-            if (obj[t] === undefined) return;
```

Comment:
This should be `if (obj[t] === undefined) continue;`

We shouldn't be setting properties to undefined. Pretty sure this was a refactoring error and that for loop used to be in a forEach.